gnmi_server/server_test.go

```diff
@@ -2738,78 +2738,6 @@ func TestClient(t *testing.T) {
     s.s.Stop()
 }
 
-func TestGnmiSetBatch(t *testing.T) {
```

Comment:
Why delete testcase?

Reply:
We need to verify performance for DASH_ROUTE_TABLE and DASH_VNET_MAPPING_TABLE.
TestGnmiSetBatch used DASH_QOS for batch test, it does not meet our requirement.